Editor/Plugins/SettingsWindow/BaseTab.cs

```diff
@@ -5,11 +5,16 @@ namespace StansAssets.Foundation.Editor
 {
     public abstract class BaseTab : VisualElement
     {
+        protected readonly VisualElement m_Root;
+
         protected BaseTab(string uxmlPath)
         {
             // Import UXML
             var visualTree = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(uxmlPath);
-            Add(visualTree.CloneTree());
+            var template = visualTree.CloneTree();
+            m_Root = template[0];
```

Comment:
`visualTree.CloneTree(this);` - Is a much better option :)

Reply:
I agree, that's a much more compact solution.
but we anyway need to cache m_Root for derived classes

Reply:
I think you are right. In this case that doesn't really matter since the next step we would do the same.

Reply:
false. we can do much better than this :)